.idea/TextMessendger.iml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Nothing from `.idea` folder shall be pushed on GitHub. Please remove these files from Pull Request